### PR TITLE
Write the communications guide

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: contributing/communications.md [hunk 1]
```diff
@@ -7,9 +7,134 @@ permalink: /contributing/communications/
 
 # Communications
 
-**:warning: This page is not complete yet :warning:**
+Hello, so you you're interested in helping out with our communications? We're pretty pleased about that. This guide covers general writing style as well as everything you need to start talking to our users on [GitHub](#github) and [Twitter](#twitter). We also cover how to [respond to Code of Conduct issues](#code-of-conduct).
```
Comment (howlingmad) [inline contributing/communications.md]: "Hello, so you you're interested in helping out" should be "Hello, so you're interested in helping out"

extra 'you' that isn't needed

Comment (rowanmanning) [inline contributing/communications.md]: Derp. Good spot

File: contributing/communications.md [hunk 2]
```diff
@@ -7,9 +7,134 @@ permalink: /contributing/communications/
...
+
+Not in a cheesy Thanksgiving "I'm thankful for" kinda way; but make sure you thank anyone who's made a contribution or is even simply _using_ Pa11y (when it feels appropriate).
+
+### ...Especially to our Contributors
+
+Our contributors are amazing – they're giving up their time to make the web as a whole more accessible. Our gratefulness should definitely come across, as long is it's not overbearing!
+
+### Even in Documentation
+
+We're not saying that the documentation should read like a conversation, that would probably be really jarring and unhelpful when decribing technical things. However introductions and any signposting should be friendly and helpful.
```
Comment (howlingmad) [inline contributing/communications.md]: decribing => describing

**Comment by howlingmad** [comment]
couple of spelling issues, otherwise 👍

**Pull request merged**